packages/cloudflare/src/cli/build/patches/ast/util.ts

```diff
@@ -50,7 +52,18 @@ export function getRuleEdits(rule: string | RuleConfig, root: SgNode, { once = f
     );
   });
 
-  return edits;
+  return { edits, matches };
+}
+
+/**
+ * Parse a file and obtain its root.
+ *
+ * @param path The file path
+ * @param lang The language to parse. Defaults to TypeScript.
+ * @returns The root for the file.
+ */
+export function parseFile(path: string, lang = Lang.TypeScript) {
```

Comment:
Nit: I would rename this function astGrepParseFile or something like that to distinguish it from the tsParseFile

(if I understand correctly you aim to completely replace/remove tsParseFile? so maybe for that reason having a more generic parseFile name here is appealing to you? if that's the case I would still recommend using astGrepParseFile until tsParseFile is still present in the codebase)

Reply:
Yep, we will move to only ast-grep.